components/SubNav/SubNav.jsx

```diff
@@ -0,0 +1,51 @@
+require('./SubNav.scss')
+
+import React from 'react'
+import Carousel from '../Carousel/Carousel'
+import StandardListItem from '../StandardListItem/StandardListItem'
+
+const tcSubNav = {
+  compete : [
+    {img: require('./placeholder.svg'), text: 'Design Challenges', link: '/challenges/design/active'},
+    {img: require('./placeholder.svg'), text: 'Development Challenges', link: '/challenges/develop/active'},
+    {img: require('./placeholder.svg'), text: 'Data Science Challenges', link: '/challenges/data/active'},
+    // TODO dynamic domain
```

Comment:
If you need the dynamic url for `arena.topcoder.com`, you should be able to use `process.env.ARENA_URL` here. We just need to add `--tc` to the npm build command.